### PR TITLE
Fixes to stabilize master

### PR DESCRIPTION
**Pull request by jbhilton11**
#67

### CODE REVIEW

File: src/cssEditor/css-editor.js [hunk 1]
```diff
@@ -16,7 +16,11 @@ export class CssEditor {
         }
     }
     
-    attached() {
+    attached(params) {
+        if (params.id) {
+          this.pastebinId = params.id;
+        }
+
```
Comment (Alaboudi1) [inline src/cssEditor/css-editor.js]: #67

File: src/htmlEditor/html-editor.js [hunk 2]
```diff
@@ -16,7 +16,11 @@ export class HtmlEditor {
         }
     }
     
-    attached() {
+    attached(params) {
+        if (params.id) {
+          this.pastebinId = params.id;
+        }
+
```
Comment (Alaboudi1) [inline src/htmlEditor/html-editor.js]: #67

File: src/jsEditor/js-editor.js [hunk 3]
```diff
@@ -18,8 +18,11 @@ export class JsEditor {
       this.pastebinId = params.id;
     } 
   }
-
-  attached() {
+  attached(params) {
+    if (params.id) {
+      this.pastebinId = params.id;
+    }
+
```
Comment (Alaboudi1) [inline src/jsEditor/js-editor.js]: #67

File: src/pastebin/pastebin.js [hunk 4]
```diff
@@ -34,19 +34,19 @@ export class Pastebin {
...
-    this.jsEditor.attached();
-    this.jsGutter.attached();
-    this.consoleWindow.attached();
-    this.htmlEditor.attached();
-    this.cssEditor.attached();
+    this.jsEditor.attached({id: this.pastebinId});
+    this.jsGutter.attached({id: this.pastebinId});
+    this.consoleWindow.attached({id: this.pastebinId});
+    this.htmlEditor.attached({id: this.pastebinId});
+    this.cssEditor.attached({id: this.pastebinId});
```
Comment (Alaboudi1) [inline src/pastebin/pastebin.js]: #67

Comment (Alaboudi1) [inline src/pastebin/pastebin.js]: Are these the changes that fixed the router bug?

**Comment by Alaboudi1** [comment]
Can you confirm that what I have[ marked ](https://github.com/tlatoza/SeeCodeRun/pull/71/files)are the only changes that fixed the bug?

**Comment by jbhilton11** [comment]
Yes I believe what you've marked is the fix for #67.

**Comment by jbhilton11** [comment]
The other fixes were in relation to the master not working correctly.

**Comment by Alaboudi1** [comment]
Cool. The code looks good to me.

**Pull request merged**